tests/recommender_base_test.py

```diff
@@ -35,6 +35,12 @@ def test_recommend(self):
         recs = model.recommend(0, user_items, N=10000)
         self.assertTrue(len(recs))
 
+        # filter recommended items using an additional filter list
```

Comment:
We know that we should get item_id = user_id as recommendation but as filtered out there should still be recommendations but not equal to the filtered object.